app/javascript/components/Statistic.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import styled from 'styled-components';
+
+const StatisticStyled = styled.div`
+  border: 1px solid #DAE1E7;
+  border-radius: 4px;
+  color: #3D4852;
+  padding: .75rem;
+  margin: .25rem;
+  margin-left: 0;
+  max-width: 500px;
+`;
+
+const RepositoryStyled = styled.p`
+  text-transform: uppercase;
+  font-weight: 700;
+  font-size: .65rem;
+  color: black;
+`;
+
+const StatisticStateStyled = styled.p`
+  text-transform: capitalize;
+`;
```

Comment:
we already have tailwind setup in this can we just bring these classes into regular css instead of using styled components? Easier to read and change later too 😄

Reply:
Completed in https://github.com/oddballio/magnifier/pull/60/commits/101ed12615c9a6683e133f906a882db937427a71